ui/src/dataLoaders/actions/dataLoaders.ts

```diff
@@ -327,7 +327,7 @@ export const createOrUpdateTelegrafConfigAsync = () => async (
     name: TelegrafPluginOutputInfluxDBV2.NameEnum.InfluxdbV2,
     type: TelegrafPluginOutputInfluxDBV2.TypeEnum.Output,
     config: {
-      urls: ['http://127.0.0.1:9999'],
+      urls: [`${window.location.origin}`],
```

Comment:
nice

Reply:
I don't think will work in cloud, we need a specify an exact url. Also will this work in a unit test for the action?